rfc6962-bis.xml

```diff
@@ -1294,10 +1294,7 @@ but it is expected there will be a variety.
       <section title="TLS Client" anchor="tls_clients">
         <section title="Receiving SCTs or inclusion proofs">
           <t>
-            TLS clients receive SCTs or inclusion proofs alongside or in certificates. TLS clients MUST implement all of the three mechanisms by which TLS servers may present SCTs (see <xref target="tls_servers"/>). TLS clients MAY also accept SCTs via the <spanx style="verb">status_request_v2</spanx> extension (<xref target='RFC6961'/>). TLS clients that support the <spanx style="verb">transparency_info</spanx> TLS extension SHOULD include it in ClientHello messages, with <spanx style="verb">extension_data</spanx> set to &lt;TBD&gt;.
-          </t>
-          <t>
-            TODO: What should the TLS client communicate in the extension_data? Version(s) of CT that it supports? Certain types of TransItem that it can handle? Whether or not it wants to gossip?
+            TLS clients receive SCTs or inclusion proofs alongside or in certificates. TLS clients MUST implement all of the three mechanisms by which TLS servers may present SCTs (see <xref target="tls_servers"/>). TLS clients MAY also accept SCTs via the <spanx style="verb">status_request_v2</spanx> extension (<xref target='RFC6961'/>). TLS clients that support the <spanx style="verb">transparency_info</spanx> TLS extension SHOULD include it in ClientHello messages, with an empty <spanx style="verb">extension_data</spanx>.
```

Comment:
Yeah.  I suggest adding...
"The TLS server SHOULD ignore the &lt;spanx style="verb"&gt;extension_data&lt;/spanx&gt; sent by the TLS client."
...inbetween the existing 2 sentences in section 7.2 (TLS Servers -> TLS Extension).